src/core/server/config/deprecation/core_deprecations.test.ts

```diff
@@ -318,4 +318,119 @@ describe('core deprecations', () => {
       expect(messages).toEqual([]);
     });
   });
+
+  describe('logging.dest', () => {
+    it('warns when dest is used', () => {
+      const { messages } = applyCoreDeprecations({
+        logging: { dest: 'stdout' },
+      });
+      expect(messages).toMatchInlineSnapshot(`
+        Array [
+          "\\"logging.dest\\" has been deprecated and will be removed in 8.0. To set the destination moving forward, you can use the \\"console\\" appender in your logging configuration or define a custom one. For more details, see https://github/elastic/kibana/blob/master/src/core/server/logging/README.mdx.",
+        ]
+      `);
+    });
+    it('warns when dest path is given', () => {
+      const { messages } = applyCoreDeprecations({
+        logging: { dest: '/log-log.txt' },
+      });
+      expect(messages).toMatchInlineSnapshot(`
+        Array [
+          "\\"logging.dest\\" has been deprecated and will be removed in 8.0. To set the destination moving forward, you can use the \\"console\\" appender in your logging configuration or define a custom one. For more details, see https://github/elastic/kibana/blob/master/src/core/server/logging/README.mdx.",
+        ]
+      `);
+    });
+  });
+
+  describe('logging.quiet, logging.silent and logging.verbose', () => {
```

Comment:
I initially applied the config deprecations for each of these individually. It made more sense to combine them because the the KP alternative uses the `level` for setting the log verbosity.